rosidl_generator_cpp/test/test_interfaces.cpp

```diff
@@ -153,77 +147,74 @@ TEST(Test_rosidl_generator_traits, has_bounded_size) {
     "PrimitivesDefault::has_bounded_size is true");
 
   static_assert(
-    rosidl_generator_traits::has_bounded_size<rosidl_generator_cpp::msg::PrimitivesStatic>::value,
-    "PrimitivesStatic::has_bounded_size is false");
-
-  static_assert(
-    rosidl_generator_traits::has_bounded_size<rosidl_generator_cpp::msg::PrimitivesBounded>::value,
-    "PrimitivesBounded::has_bounded_size is false");
+    !rosidl_generator_traits::has_bounded_size<rosidl_generator_cpp::msg::Primitives>::value,
```

Comment:
I'd like to point out that before we expected this message to have bounded size. But I think since we [introduced a field with type `string`](https://github.com/ros2/test_interface_files/blob/41c8c697dab6b2c7254aa0cfe051842e0e3a8bd4/msg/Primitives.msg#L14), it no longer has bounded size.

Reply:
Would it make sense to keep the original message and continue to test that the complete message is bounded? I guess the test wants to make sure that a message composed out of bounded fields is still bounded.